modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -107,4 +105,82 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-180358. Verify all status changes after 'need more info' request"
+	@priority = 5
+	test VerifyStatusAfterRequestMoreInfo {
```

Comment:
please validate with Heidi her new macro to perform this organization creation flow

Reply:
The flow of my test is different from the macro it is creating, I took this doubt with it.